templates/Deepcell_Postprocessing.ipynb

```diff
@@ -34,22 +34,32 @@
    "outputs": [],
    "source": [
     "# set up file paths\n",
-    "base_dir = \"../data/example_dataset/\"\n",
+    "base_dir = os.path.join(\"..\", \"data\", \"example_dataset\")\n",
```

Comment:
any reason this was changed? I feel like the former is easier for lab users to edit.  If it's for os compatibility (mac/windows/linux), I think there's another way to do this s.t posix-like paths are interoperable with dos-like paths (maybe thru pathlib?)

Reply:
s.t = such that 

(math major habit)

Reply:
I can change it back to what it was earlier.